stagemonitor-core/src/main/java/org/stagemonitor/core/elasticsearch/ElasticsearchClient.java

```diff
@@ -64,7 +64,8 @@ public ElasticsearchClient(final CorePlugin corePlugin, final HttpClient httpCli
 			PooledResourceMetricsRegisterer.registerPooledResource(pooledResource, Stagemonitor.getMetric2Registry());
 		}
 		this.httpClient = httpClient;
-		if (esAvailabilityCheckIntervalSec > 0) {
```

Comment:
I think if you don't provide a elasticsearch url, the client wont be instantiated. So yes, I would just remove the condition.